ui/scss/init/_vars.scss

```diff
@@ -11,7 +11,7 @@
   --height-radio: 24px;
   --height-badge: 24px;
 
-  // Spacing
+  // Spacing (rem is 16px)
```

Comment:
This is always relative, so it's an unnecessary comment.  Even if it's hardcoded, the comment will just make it a maintenence burden and pointless